src/main/resources/com/amazonaws/codedeploy/AWSCodeDeployPublisher/help-applicationName.html

```diff
@@ -1,7 +1,7 @@
 <div>
   The name of the AWS CodeDeploy application you wish to deploy to. This plugin assumes that you've
   already created the application and deployment group. If you haven't already, work through the <a
-  href="http://alpha-docs-aws.amazon.com/codedeploy/latest/userguide/how-to-create-application.html">How
+  href="http://docs.aws.amazon.com/codedeploy/latest/userguide/how-to-create-application.html">How
```

Comment:
I think you want to use https to access that page.
```suggestion
  href="https://docs.aws.amazon.com/codedeploy/latest/userguide/how-to-create-application.html">How
```